client/src/components/Navbar/Menus/Unauthenticated/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Grid, makeStyles } from '@material-ui/core';
+import { useHistory } from 'react-router';
+import { Link } from 'react-router-dom';
+
+const useStyles = makeStyles({
+  button: {
+    color: 'white',
+    fontWeight: 'bolder',
+    border: '1px solid white',
+    padding: '15px 40px',
+    borderRadius: 0,
+    letterSpacing: 1,
+    textDecoration: 'none',
+  },
+});
+
+export const index = (): JSX.Element => {
+  const classes = useStyles();
+  const history = useHistory();
+  const [location, setLocation] = React.useState(history.location.pathname);
```

Comment:
this doesn't really need to be a state you update. `history.location.pathname` will already be updated. If you just want to save typing `history.location.pathname` multiple times, you can just use a normal variable like `const location = history.location.pathname`

Reply:
Could we review this together? The reason `history` doesn't currently change is because the `Navbar` component is mounted in `App.tsx` and doesn't get updated unless in a `React.useState` to ensure re-renders. Check out this video [here](https://www.loom.com/share/dfd4bba33fe24cf7a0847f5cee1bd195)